packages/guui/components/Dropdown/index.js

```diff
@@ -219,7 +228,9 @@ export default class Dropdown extends Component<
                 ) : (
                     <button
                         onClick={this.toggle_}
-                        className={this.state.isExpanded ? 'expanded' : ''}
+                        className={
```

Comment:
As per my previous comment, this looks like a use case for using `cx` to conditionally render class names.